oxenss/server/omq.h

```diff
@@ -130,8 +130,8 @@ class OMQ {
     ///   - P -- an ed25519 pubkey underlying a session ID, in bytes (32 bytes).  The account
     ///     will be derived by converting to an x25519 pubkey and prepending the 0x05 byte.  The
     ///     signature uses *this* key, not the derived x25519 key.
-    /// - S -- (optional) a 32-byte authentication subkey to use for authentication.  The
-    ///   signature with such a subkey uses a derived key (as described in the RPC endpoint
+    /// - S -- (optional) a 32-byte authentication subkey tag to use for authentication.  The
```

Comment:
this sounds a bit more ambiguous as "tags" are also used in authenticated encryption as authentication tags.

Reply:
True, but the next line is "go see the rpc docs" which has a paragraph about the subkey value.